calico-enterprise_versioned_docs/version-3.19-1/releases.json

```diff
@@ -1,247 +1,269 @@
 [
   {
-    "title": "master",
+    "title": "v3.19.0-1.0",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.31.2",
```

Comment:
I think this will be v1.33.0 
cc: @danudey

Reply:
Fixed, but ping me of not correct.